user/advanced-topics/usb-qubes.md

```diff
@@ -95,26 +102,67 @@ sys-usb dom0 allow
 
 (Change `sys-usb` to your desired USB qube.)
 
-You can now use your USB keyboard to login and for LUKS decryption during boot.
+You can now use your USB keyboard to log in and for LUKS decryption during
+boot.
```

Comment:
The above does not cover LUKS passphrase prompt (sys-usb is not running at this point yet). This one is covered by removing `rd.qubes.hide_all_usb` option from the kernel cmdline.